.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 services:
 - docker
 python:
-- '3.6'
+- '3.7.5'
```

Comment:
Ideally we want to upgrade the python version across the project if we decide to bump up in travis for the sake of consistency. 

Can you please upgrade in the docker requirements and see if it works? I remember running into some issue when upgrading the docker python to 3.7.5

Reply:
Hi! I just received a notification an hour ago because of the bot. I did not know you commented here. Let me check that out.

Reply:
I just pulled the master branch with the latest commit, and `docker-compose --build` does not work. I am thrown this error,
```shell
Step 2/6 : RUN apt-get update &&   apt-get install -q -y default-jdk
 ---> Running in 379e2b43bde4
ERROR: Service 'worker' failed to build: failed to create endpoint elated_tharp on network bridge: failed to add the host (veth6d1a8f2) <=> sandbox (veth7e6d7eb) pair interfaces: operation not supported
```

I'm assuming there is something wrong with the package `default-jdk`, but I'm not that professional with docker so I maybe wrong. Please let me know when the master branch builds again so I can test with the new version number for Python.